lib/metadata/linux/MiqRpmPackages.rb

```diff
@@ -99,9 +103,18 @@ def close
     end
     
     private
+
+    def time_tag?(tag)
+      tag == "installtime"
+    end
+
+    def convert(tag, val)
+      time_tag?(tag) ? Time.at(val) : val
```

Comment:
We should ensure that UTC time objects make their way into the database, so I think this should change to `Time.at(val).utc`.  However, perhaps this is not the right place and there is a better place to ensure that UTC conversion.